test/truffle/integration/startup.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+source test/truffle/common.sh.inc
+
+jt ruby -J-verbose:gc -e 'p :hi' > temp.txt
```

Comment:
Can we write this out to `/tmp` instead? Or honor the `SPEC_TEMP_DIR` env var? I'd like to avoid unnecessary writes to my SSD if possible. `/tmp` uses tmpfs by default on Linux and thus all writes are to RAM.

Reply:
Nevermind. I missed that this was an integration test and thus isn't run locally all that frequently.

Reply:
Yeah, all integration tests follow this pattern.
We could easily change it if it matters, by having a `$OUTPUT_FILE` defined in `common.sh.inc` or so.
But a file in the current dir is convenient to debug.